crates/storage/codecs/src/alloy/log.rs

```diff
@@ -1,6 +1,6 @@
 //! Native Compact codec impl for primitive alloy log types.
 
-use crate::Compact;
+use crate::{Compact, Vec};
```

Comment:
not a fan of importing Vec like this.

I'm fine with

#[cfg(not(feature = "std"))]
use alloc::vec::Vec;

Reply:
Fixed